static/app/views/performance/transactionSummary/pageLayout.tsx

```diff
@@ -95,43 +184,55 @@ function PageLayout(props: Props) {
             forceProject={project}
             specificProjectSlugs={defined(project) ? [project.slug] : []}
           >
-            <StyledPageContent>
-              <NoProjectMessage organization={organization}>
-                <TransactionHeader
-                  eventView={eventView}
-                  location={location}
-                  organization={organization}
-                  projects={projects}
-                  projectId={projectId}
-                  transactionName={transactionName}
-                  currentTab={tab}
-                  hasWebVitals={tab === Tab.WebVitals ? 'yes' : 'maybe'}
-                  onChangeThreshold={(threshold, metric) => {
-                    setTransactionThreshold(threshold);
-                    setTransactionThresholdMetric(metric);
-                  }}
-                  metricsCardinality={metricsCardinality}
-                />
-                <Layout.Body>
-                  {defined(error) && (
-                    <StyledAlert type="error" showIcon>
-                      {error}
-                    </StyledAlert>
-                  )}
-                  <ChildComponent
+            <Tabs
+              value={tab}
+              onChange={newTab => {
+                // Prevent infinite rerenders
+                if (newTab === tab) {
+                  return;
+                }
+                trackTabClick(newTab);
+                browserHistory.push(getNewRoute(newTab));
+              }}
```

Comment:
is Tabs expensive to re-render because of the nested children? I'd maybe `useCallback`

Reply:
I don't think it's expensive, but still did so in https://github.com/getsentry/sentry/pull/39766/commits/3eecc58f60d6cb47fc4b49fcdb9301f941b3749c